assets/fonts/gigaspaces/style.css

```diff
@@ -1,217 +1,173 @@
 @font-face {
-	font-family: 'gigaspaces';
-	src:url('fonts/gigaspaces.eot?-sbgnzc');
-	src:url('fonts/gigaspaces.eot?#iefix-sbgnzc') format('embedded-opentype'),
-		url('fonts/gigaspaces.woff?-sbgnzc') format('woff'),
-		url('fonts/gigaspaces.ttf?-sbgnzc') format('truetype'),
-		url('fonts/gigaspaces.svg?-sbgnzc#gigaspaces') format('svg');
-	font-weight: normal;
-	font-style: normal;
+  font-family: 'gigaspaces';
+  src:  url('fonts/gigaspaces.eot?jz26tz');
+  src:  url('fonts/gigaspaces.eot?jz26tz#iefix') format('embedded-opentype'),
+    url('fonts/gigaspaces.ttf?jz26tz') format('truetype'),
+    url('fonts/gigaspaces.woff?jz26tz') format('woff'),
+    url('fonts/gigaspaces.svg?jz26tz#gigaspaces') format('svg');
+  font-weight: normal;
+  font-style: normal;
 }
 
-[class^="icon-gs-"], [class*=" icon-gs-"] {
-	font-family: 'gigaspaces';
-	speak: none;
-	font-style: normal;
-	font-weight: normal;
-	font-variant: normal;
-	text-transform: none;
-	line-height: 1;
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'gigaspaces' !important;
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
 
-	/* Better Font Rendering =========== */
-	-webkit-font-smoothing: antialiased;
-	-moz-osx-font-smoothing: grayscale;
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-gs-node-status-alert:before {
-	content: "\e629";
+.icon-uniE600:before {
+  content: "\e600";
 }
-
-.icon-gs-node-status-success:before {
-	content: "\e62a";
+.icon-uniE601:before {
+  content: "\e601";
 }
-
-.icon-gs-node-status-fail:before {
-	content: "\e62b";
+.icon-uniE602:before {
+  content: "\e602";
 }
-
-.icon-gs-node-status-loading:before {
-	content: "\e630";
+.icon-uniE603:before {
+  content: "\e603";
 }
-
-.icon-gs-workflow-cancelled:before {
-	content: "\e600";
+.icon-uniE604:before {
+  content: "\e604";
 }
-
-.icon-gs-workflow-failed:before {
```

Comment:
This classes are used in EventUtils in cloudify-stage: https://github.com/cloudify-cosmo/cloudify-stage/blob/1a2a27a92ac754e8da93598d1c1a2443c484f5b0/widgets/common/src/EventUtils.js#L33

Reply:
Oh, it removed the names also..
will fix it